.github/workflows/documentation.yml

```diff
@@ -8,6 +8,9 @@ on:
         required: true
         default: 'release-xxxxxxxx'
 
+permissions:
+  contents: read  #  to fetch code (actions/checkout)
```

Comment:
The wiki job also pushes to a repository.

Reply:
I see, that makes sense. So it doesn't need write permission wherever a `token` is provided for the checkout action.

Reply:
Yes, the `permissions` section applies only to the auto generated token. When you use a secret token in checkout action it is saved on disk in the action and is reused later when you do `git push` from that directory.